lib/Doctrine/DBAL/Platforms/PostgreSQL100Platform.php

```diff
@@ -16,6 +16,9 @@ protected function getReservedKeywordsClass() : string
         return PostgreSQL100Keywords::class;
     }
 
+    /**
+     * {@inheritDoc}
```

Comment:
What exactly do we need to explicitly inherit?

Reply:
the type information

Reply:
I already brought it up somewhere but [`{@inheritDoc}`](https://docs.phpdoc.org/latest/guides/inheritance.html#the-inheritdoc-tag) is irrelevant to the type information which is [inherited automatically](https://docs.phpdoc.org/latest/guides/inheritance.html#methods). While this may be a known limitation of the slevomat/doctrine standard, I don't like this turning into a cargo cult.

Reply:
> Currently some applications have DocBlocks containing just the {@inheritDoc} inline tag to indicate that their complete contents should be inherited. This usage breaks with the PHPDoc Standard as summaries cannot contain inline tags and inheritance is automatic; you do not need to define a special tag for it.

If I understand this correctly, this should never happen:
```
/**
 * {@inheritDoc}
 */
```

> However, it does make clear that an element has been explicitly documented (and thus not forgotten). As such we are working to include a new (normal) tag in the PHPDoc Standard @inheritDoc that will serve that purpose.

but this can:

```
/**
 * @inheritDoc
 */
```

The former happens all over the place in this lib… maybe we should merge as is and this should fix afterwards? I reported the issue here: https://github.com/slevomat/coding-standard/issues/991

Reply:
Weirdly, the build currently fails because of extra `{@inheritDoc}`: https://travis-ci.org/github/doctrine/dbal/jobs/682337729#L242-L243 Something is off…

Reply:
> […] maybe we should merge as is and this should fix afterwards?

Yeah. Just wanted to make sure we're not blindly adding some annotations and removing exactly the same ones in other places just to pass the check.

Reply:
> Something is off…

I just needed to rebase

Reply:
@morozov as suggested in https://github.com/slevomat/coding-standard/issues/991#issuecomment-623069823 , I think we should wrap this up and merge as is.

Reply:
Agree. We can keep the discussion in the above issue.